### ISSUE TITLE
Is this maintained / being worked on? Basic issues with startup

### ISSUE THREAD

**Issue by jd-bus** [daily-demos/daily-flutter-demo#4]
Hi,

I wanted to ask if this still being worked on / maintained, or if it's on plan to launch daily flutter more generally.

When I run this project I get the following error:

```
    [!] Invalid `Podfile` file: undefined method `exists?' for File:Class.

     #  from /Users/jim/fltr/daily-flutter-demo/ios/Podfile:34
     #  -------------------------------------------
     #  
     >    if File.exists?('DailyFlutter-iOS.podspec')
     #      pod 'DailyFlutter-iOS', :podspec => 'DailyFlutter-iOS.podspec'
     #  -------------------------------------------
```

Here is my cocaopods version:

```
% pod --version
1.12.1
```

In researching the error, I see the following answer:
https://stackoverflow.com/questions/75414034/invalid-podfile-file-undefined-method-exists-for-fileclass
"Downgrade cocoapods version can fix it. But if you use XCode 14.3++ and want to archive app, you will get issues."

In other words, daily-flutter-demo does not worth with modern versions of cocaopods, and therefore does not work with modern versions of XCode.

More generally, the Daily docs (https://docs.daily.co/reference/flutter) refer to Flutter support as "pre-beta" but this repo hasn't been touched in several months, so I'd really appreciate any frank feedback on whether it's truly pre-beta or just abandoned / not currently being worked on.  Thanks so much!

**Comment by filipi87** [comment]
Hi @jd-bus, yes, we released a new version of `daily_flutter: ^0.16.0-beta.1` yesterday. 

This demo has been updated to utilize the latest version, ensuring compatibility with recent versions of Xcode. This update also addresses the issue you mentioned above.

Thanks for reporting this.

**Comment by jd-bus** [comment]
ok thank you.  now facing this issue when try to use latest with android. do you know about it?

```
> A failure occurred while executing com.android.build.gradle.internal.tasks.CheckDuplicatesRunnable
   > Duplicate class org.webrtc.AddIceObserver found in modules jetified-android-114.5735.08.1-runtime (io.github.webrtc-sdk:android:114.5735.08.1) and jetified-daily_flutter_android_platform_components-0.16.0-beta.1-runtime (co.daily:daily_flutter_android_platform_components:0.16.0-beta.1)
     Duplicate class org.webrtc.AndroidVideoDecoder found in modules jetified-android-114.5735.08.1-runtime (io.github.webrtc-sdk:android:114.5735.08.1) and jetified-daily_flutter_android_platform_components-0.16.0-beta.1-runtime (co.daily:daily_flutter_android_platform_components:0.16.0-beta.1)
     Duplicate class org.webrtc.AndroidVideoDecoder$1 found in modules jetified-android-114.5735.08.1-runtime (io.github.webrtc-sdk:android:114.5735.08.1) and jetified-daily_flutter_android_platform_components-0.16.0-beta.1-runtime (co.daily:daily_flutter_android_platform_components:0.16.0-beta.1)
```

**Comment by filipi87** [comment]
Hi @jd-bus, have you tried to clean everything before trying to build the app again ?

Or please let me know the steps that you are executing so I can try to reproduce this issue.
